index.html

```diff
@@ -72,9 +70,209 @@ <h3 class="discription-header">I'm a Software Developer</h3>
+            </p>
+                <ul class="work-tag-list">
+                    <li class="work-tag-items">html</li>
+                    <li class="work-tag-items">bootstrap</li>
+                    <li class="work-tag-items">Ruby</li>
+                </ul>
+                
+                </div>
+                <div class='work-button-section'>
+          <a class='card-button'>See Project</a>
+                </div>
+            </div>
+
+            <div class="work-card">
+                <div class="info-card-holder">
+                    <h2 class="work-project-header">Profesional Art Printing Data
+                    </h2>
+                    <p class="work-card-description"> A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry' standard.
+            </p>
+                <ul class="work-tag-list">
+                    <li class="work-tag-items">html</li>
+                    <li class="work-tag-items">bootstrap</li>
+                    <li class="work-tag-items">Ruby</li>
+                </ul>
+                
+                </div>
+                <div class='work-button-section'>
+          <a class='card-button'>See Project</a>
+                </div>
+            </div>
+
+            <div class="work-card">
+                <div class="info-card-holder">
+                    <h2 class="work-project-header">Profesional Art Printing Data
+                    </h2>
+                    <p class="work-card-description"> A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry' standard.
+            </p>
+                <ul class="work-tag-list">
+                    <li class="work-tag-items">html</li>
+                    <li class="work-tag-items">bootstrap</li>
+                    <li class="work-tag-items">Ruby</li>
+                </ul>
+                
+                </div>
+                <div class='work-button-section'>
+          <a class='card-button'>See Project</a>
+                </div>
+            </div>
+           
+        </div>
+    </section>
+
+     <!-- About me section-->
+     <section class="about" id="about">
+        <div class="about-me-section">
+            <div class="about-me-sub-section">
+                <h2 class="about-me-header">About me</h2>
+                <p class="about-me-description">
+                    Hello I'm a software developer! I can help you build a product ,
+  feature or website Look through some of my work and experience! If
+  you like what you see and have a project you need coded, don't
+  hestiate to contact me.
+                </p>
+                <button type="button" class="my-resume-button">Get My Resume</button>
+            </div>
+            <div class="about-illustration">
+                <img src="./images/background-2.png" alt="img" />
+            </div>
```

Comment:
- [ ] Excellent job including your gym post image here 🚀. However, please kindly ensure you include the file `background-2.png` inside your `images` folder 🤓.
    >*Hint : I think I saw the missing image inside another branch called kiko. 🔧

This is how your design looks 👇🏽

![design](https://user-images.githubusercontent.com/116701485/235292513-43405110-d257-49cc-8398-ec6a0705db26.png)